driver/ppm.h

```diff
@@ -41,6 +41,12 @@ or GPL2.txt for full copies of the license.
 #define PPM_PORT_STATSD 8125
 #define PPM_PORT_MONGODB 27017
 
+#if (LINUX_VERSION_CODE >= KERNEL_VERSION(5, 6, 0))
```

Comment:
A cleaner (though more time-intensive) approach would be to abstract these functions behind a consistent API. Ultimately, we put a `uint64_t` on the ring buffer, and I don't think there's any reason why we can't just work in nanoseconds throughout the driver rather than storing timespec structs.